generator.yaml

```diff
@@ -19,6 +19,10 @@ resources:
         - path: Status.State
           in: [ "Active" ]
     fields:
+      CodeS3SHA256:
+        type: String
+        compare:
+          is_ignored: true
```

Comment:
@Vandita2020 I'm struggling to understand why there is a need for an additional `Spec.CodeS3SHA256` field here when there is already a `Status.CodeSHA256` field that can be used to compare checksums with?

Reply:
We have added this field in our `Spec` to see if the code has changed. Since the Function APIs don't store the current `S3.Bucket` or `S3.Key`, there is no way to identify if the change has made when the user changes these fields, leading to no update for the deployment package. `SHA256` helps identifying the update, given the user has to provide the updated `SHA256` along with the new deployment package.